### PR TITLE
fix: Fix task runner default n8n uri and auth endpoint (no-changelog)

### PR DESCRIPTION
**Pull request by tomi**
## Summary

Fix task runner default n8n uri and auth endpoint

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/@n8n/task-runner/src/start.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ function readAndParseConfig(): Config {
 	}
 
 	return {
-		n8nUri: process.env.N8N_RUNNERS_N8N_URI ?? 'localhost:5678',
+		n8nUri: process.env.N8N_RUNNERS_N8N_URI ?? 'localhost:5679',
```
Comment (valya) [inline packages/@n8n/task-runner/src/start.ts:23]: Shouldn't this be 127.0.0.1 now?

Comment (tomi) [reply packages/@n8n/task-runner/src/start.ts:23]: Yes it should. Fixing

**Review by valya — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.63.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.63.0)

**Comment by janober** [comment]
Got released with [`n8n@1.63.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.63.0)

**Pull request merged**